README.md

```diff
@@ -114,6 +112,8 @@ The `imaging` schema stores information regarding the calcium traces for each ma
 
      + If your workflow uses these packages, you should install them when you install `element-interface`.
 
+     <details>
+     <summary>Click to expand details</summary>
```

Comment:
I looks like backtic formatting isn't working properly in some cases. Adding an extra line between `</summary>` and `+ Bulleted item` seems to help. 

Current screenshot:
> ![Screen Shot 2022-04-26 at 09 09 57](https://user-images.githubusercontent.com/9404707/165322267-cdba4c63-7733-4958-b7e5-bf0a62a63141.png)

With extra line:
> ![Screen Shot 2022-04-26 at 09 23 44](https://user-images.githubusercontent.com/9404707/165322292-88ada256-9d08-4a97-b2bc-80ed9a1d0178.png)

Reply:
Good catch!  I have fixed the four locations with this issue.